docs/build/build-protocol-info.md

```diff
@@ -41,16 +41,15 @@ Polkadot (and Substrate) use the SS58 address format. This is a broad "meta-form
 handle many different cryptographic schemes and chains. It has much in common with Bitcoin's
 Base58Check format such as a version prefix, a hash-based checksum suffix, and base-58 encoding.
 
-See the [SS58 page](https://docs.substrate.io/v3/advanced/ss58/) in the Substrate documentation for
+See the [SS58 page](https://docs.substrate.io/main-docs/fundamentals/accounts-addresses-keys/#address-encoding-and-chain-specific-addresses) in the Substrate documentation for
 encoding information and a more comprehensive list of network prefixes.
 
 :::warning Do not use regular expressions (regex) to validate addresses
 
 Always verify using the
 prefix and checksum of the address. Substrate API Sidecar provides an
 `accounts/{accountId}/validate` path that returns a boolean `isValid` response for a provided
-address. If you want to validate addresses in another way, see the
-[validation documentation](https://docs.substrate.io/v3/advanced/ss58/#validating-addresses).
```

Comment:
Are the alternative ways to validate addresses no longer listed on Substrate docs?